src/main/java/org/bihealth/mi/easybus/Message.java

```diff
@@ -65,4 +73,45 @@ public Participant getRespondTo(){
     public String getReplyID(){
         return this.replyID;
     }
+    
+    /**
+     * Serializes this messages to string
+     * 
+     * @return
+     * @throws IOException
+     */
+    public String serialize() throws IOException {
+        return serializeMessage(this);
+    }
+    
+    /**
+     * Serialize message as base64 encoded string
+     *
+     * @param msg the msg
+     * @return the string
+     * @throws IOException Signals that an I/O exception has occurred.
+     */
+    public static String serializeMessage(Message msg) throws IOException {
```

Comment:
Das Konstrukt mit serialize und serializeMessage verstehe ich nicht ganz. Du rufst serializeMessage dann auch aus dem Fragment auf. Müsste es nicht funktionieren, den Code aus serializeMessage direkt in serialize zu packen und dann funktioniert das auch in der Subklasse? Oder du überschreibst serialize() in der Subklasse und rufst dann super.serialize() auf?

Wenn das nicht geht, dann könnte zumindest serializeMessage(...) protected statt public sein,

Reply:
Code in serialize() umgezogen und serializeMessage() entfernt